GetStream/stream-chat-android
# Fix refreshing channels list when syncing the channel

closes #3257 

### 🎯 Goal

Fix refreshing channels list when syncing the channel.

### 🛠 Implementation details

`QueryChannelsLogic` shouldn't try to refresh the channel if it's not present in the memory yet as it might be not populated with data.

### 🎨 UI Changes
<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/17440581/167579906-721614e1-98d8-4f11-8ca6-480c1c4108f3.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/17440581/167579868-5156358a-7fb5-47a2-a9b4-b9aabde9a537.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>


### 🧪 Testing
1. Turn on airplane mode and send a message in the channel
2. Kill the app
3. Open the app and stay on the channels list
4. Turn off airplane mode
5. Observe channels list

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
